### ISSUE TITLE
Svelte 0.13.0 and Webstorm 2020.2.1: SvelteFoldingBuilder error

### ISSUE THREAD

**Issue by ssanbeg** [tomblachut/svelte-intellij#155]
My plugins are up-to-date as are my IDEs. Your plugin started crashing this morning. I have restarted the IDE, I've Invalidated Cache/Restart and the below message continues to show up. MacBook Pro, 2019, running Catalina 10.15.6, fully patched.

```
java.lang.IllegalStateException: super.getLanguagePlaceholderText(node, range) must not be null
	at dev.blachut.svelte.lang.editor.SvelteFoldingBuilder.getLanguagePlaceholderText(SvelteFoldingBuilder.kt:56)
	at com.intellij.lang.XmlCodeFoldingBuilder.addToFold(XmlCodeFoldingBuilder.java:213)
	at com.intellij.lang.XmlCodeFoldingBuilder.doAddForChildren(XmlCodeFoldingBuilder.java:102)
	at dev.blachut.svelte.lang.editor.SvelteFoldingBuilder.doAddForChildren(SvelteFoldingBuilder.kt:22)
	at com.intellij.lang.XmlCodeFoldingBuilder.addElementsToFold(XmlCodeFoldingBuilder.java:80)
	at com.intellij.lang.XmlCodeFoldingBuilder.doAddForChildren(XmlCodeFoldingBuilder.java:90)
	at dev.blachut.svelte.lang.editor.SvelteFoldingBuilder.doAddForChildren(SvelteFoldingBuilder.kt:22)
	at com.intellij.lang.XmlCodeFoldingBuilder.addElementsToFold(XmlCodeFoldingBuilder.java:80)
	at com.intellij.lang.XmlCodeFoldingBuilder.doAddForChildren(XmlCodeFoldingBuilder.java:90)
	at dev.blachut.svelte.lang.editor.SvelteFoldingBuilder.doAddForChildren(SvelteFoldingBuilder.kt:22)
	at com.intellij.lang.XmlCodeFoldingBuilder.addElementsToFold(XmlCodeFoldingBuilder.java:80)
	at com.intellij.lang.XmlCodeFoldingBuilder.doAddForChildren(XmlCodeFoldingBuilder.java:90)
	at dev.blachut.svelte.lang.editor.SvelteFoldingBuilder.doAddForChildren(SvelteFoldingBuilder.kt:22)
	at com.intellij.lang.XmlCodeFoldingBuilder.buildLanguageFoldRegions(XmlCodeFoldingBuilder.java:74)
	at com.intellij.lang.folding.CustomFoldingBuilder.buildFoldRegions(CustomFoldingBuilder.java:44)
	at com.intellij.lang.folding.LanguageFolding.buildFoldingDescriptorsNoPlaceholderCaching(LanguageFolding.java:92)
	at com.intellij.lang.folding.LanguageFolding.buildFoldingDescriptors(LanguageFolding.java:75)
	at com.intellij.codeInsight.folding.impl.FoldingUpdate.getFoldingsFor(FoldingUpdate.java:266)
	at com.intellij.codeInsight.folding.impl.FoldingUpdate.getFoldingsFor(FoldingUpdate.java:233)
	at com.intellij.codeInsight.folding.impl.CodeFoldingManagerImpl.buildInitialFoldings(CodeFoldingManagerImpl.java:134)
	at com.intellij.openapi.fileEditor.impl.text.PsiAwareTextEditorImpl.loadEditorInBackground(PsiAwareTextEditorImpl.java:52)
	at com.intellij.openapi.fileEditor.impl.text.AsyncEditorLoader.lambda$scheduleLoading$0(AsyncEditorLoader.java:95)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:744)
	at com.intellij.openapi.fileEditor.impl.text.AsyncEditorLoader.lambda$scheduleLoading$1(AsyncEditorLoader.java:93)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.insideReadAction(NonBlockingReadActionImpl.java:508)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.lambda$attemptComputation$3(NonBlockingReadActionImpl.java:456)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1110)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runInReadActionWithWriteActionPriority$0(ProgressIndicatorUtils.java:77)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runActionAndCancelBeforeWrite(ProgressIndicatorUtils.java:153)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runWithWriteActionPriority$1(ProgressIndicatorUtils.java:114)
	at com.intellij.openapi.progress.ProgressManager.lambda$runProcess$0(ProgressManager.java:56)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:170)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:629)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:581)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:157)
	at com.intellij.openapi.progress.ProgressManager.runProcess(ProgressManager.java:56)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runWithWriteActionPriority(ProgressIndicatorUtils.java:111)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runInReadActionWithWriteActionPriority(ProgressIndicatorUtils.java:77)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.attemptComputation(NonBlockingReadActionImpl.java:473)
	at com.intellij.openapi.application.impl.NonBlockingReadActionImpl$Submission.lambda$transferToBgThread$1(NonBlockingReadActionImpl.java:397)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:215)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:26)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:194)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:207)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:183)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by tomblachut** [comment]
interesting, thx for reporting

**Comment by tomblachut** [comment]
Fix released as part of 0.17.*

**Comment by ssanbeg** [comment]
Thank you.
